### PR TITLE
Added first_map_only parameter in docs

### PR DESCRIPTION
**Pull request by hardesh**
Adresses #281

### CODE REVIEW

File: configuration/packages/configuring-amcl.rst [hunk 1]
```diff
@@ -467,6 +467,18 @@ Parameters
   Description
     Map topic to subscribe to.
 
+:first_map_only_:
```
Comment (SteveMacenski) [inline configuration/packages/configuring-amcl.rst:470]: No underscore, then LGTM!

File: configuration/packages/configuring-amcl.rst [hunk 2]
```diff
@@ -513,6 +525,7 @@ Example
         map_topic: map
         set_initial_pose: false
         always_reset_initial_pose: false
+        first_map_only_: false
```
Comment (SteveMacenski) [inline configuration/packages/configuring-amcl.rst:528]: Remove the underscore here too!

**Review by SteveMacenski — CHANGES_REQUESTED** [review]

File: configuration/packages/configuring-amcl.rst (hunk 1 above)
Comment (hardesh) [reply configuration/packages/configuring-amcl.rst:470]: Hi Steve,
I think we'll have to change it in the code as well https://github.com/ros-planning/navigation2/blob/0a961df9cbe50356484877681abfdbeae6ba7b33/nav2_amcl/src/amcl_node.cpp#L1058

Comment (hardesh) [reply configuration/packages/configuring-amcl.rst:470]: only this parameter has underscore after it

Comment (SteveMacenski) [reply configuration/packages/configuring-amcl.rst:470]: https://github.com/ros-planning/navigation2/blob/6522fed52926f240736d628d28c11a0be8f238cb/nav2_amcl/src/amcl_node.cpp#L1058

oof, yeah that would be good to correct, thanks for bringing that up. Yes, we should remove the underscore here. That was clearly a copy-paste error when it was being ported from ROS 1. Thanks for mentioning that!

**Comment by SteveMacenski** [comment]
Any update?

**Comment by hardesh** [comment]
Should I also open a PR in nav2? I thought you were going to take this up :grimacing: 

Also if I fix this it'll be merged in `main` and it can't be backported to foxy right?

**Comment by hardesh** [comment]
I can open a PR in nav2 tomorrow

**Comment by SteveMacenski** [comment]
Yeah, opening a PR would be great! 

And no, unfortunately it can’t be backported since it would change behavior suddenly

**Comment by SteveMacenski** [comment]
@hardesh Thanks for the PRs and pointing out this issue, it is much appreciated! :small_airplane:

**Comment by hardesh** [comment]
No problem! :smiley:

**Pull request merged**